fe/fe-core/src/main/java/org/apache/doris/nereids/trees/expressions/Divide.java

```diff
@@ -50,4 +51,9 @@ public <R, C> R accept(ExpressionVisitor<R, C> visitor, C context) {
     public AbstractDataType inputType() {
         return NumericType.INSTANCE;
     }
+
+    @Override
+    public boolean nullable() throws UnboundException {
+        return true;
```

Comment:
Could u add some comments to explain why it is nullable? I guess the reason is some string cannot be cast to date type. If this is true all date related functions are nullable.

Reply:
just because be implement like this, and be do not follow fe's plan